stable/traefik/README.md

```diff
@@ -91,6 +91,7 @@ The following table lists the configurable parameters of the Traefik chart and t
 | `serviceType`                          | A valid Kubernetes service type                                                                                              | `LoadBalancer`                                    |
 | `loadBalancerIP`                       | An available static IP you have reserved on your cloud platform                                                              | None                                              |
 | `startupArguments`                       | A list of startup arguments which are passed to traefik                                                              | `[]`                                              |
+| `extraEnv`                             | Additional environment variables to set on the container                                                                     | `{}`                                              |
```

Comment:
I like the PR, and it looks good, Im just not sure `extraEnv` is the name I would use for this..

How about:
- additionalEnvVars
- customEnvVars
- envVars

I'm just not sure the `extra` portion adds anything... (and I personally like envVars as the name, but I could be convinced)

Thoughts?

Reply:
I hadn't put much thought into the naming. It seems the majority of charts in the repo are using `extraEnv` for this purpose so I figured I'd stick with that.

There are 2 charts using `envVars` currently.

In thinking about it now I do prefer all of the suggestions you made over `extraEnv`. If you're not fussed on being different from the other charts then I can change the name to `envVars`.